lib/common/client/middleware/faraday_middleware_patch.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module FaradayMiddlewarePatch
+  def initialize(app = nil, options = {})
+    @app = app
+    @options = @@default_options.merge(options)
```

Comment:
I wonder if we should use a class reader here. What happens if we omit setting the var in the config/initializer? Does this try to call merge on nil?

Reply:
Good catch on testing without the initializer. I tweaked slightly to add a default value, as I couldn't get a reader to work for all cases (some weirdness with scope thanks to the monkeypatch `prepend` syntax, I think)